pyartifactory/models/auth.py

```diff
@@ -13,6 +13,7 @@ class AuthModel(BaseModel):
 
     url: str
     auth: Tuple[str, SecretStr]

```

Comment:
```suggestion
    auth: Optional[Tuple[str, SecretStr]] = None
```
else the auth parameter is still mandatory

Reply:
You will also need to change the main class constructor to add the `access_token` parameter and make `auth` optional https://github.com/anancarv/python-artifactory/blob/master/pyartifactory/objects/artifactory.py#L20

Reply:
Done